demo-shell/src/app.config.json

```diff
@@ -1009,6 +1009,9 @@
     "expandedSidenav": true,
     "preserveState": true
   },
+  "adf-cloud-task-filters": {
+    "ws-notifications": true
```

Comment:
I think that we should have a ws-notification true false across components. right now task list is the only one that uses it but as next task we will have the same for process.

Reply:
@eromano @popovicsandras @DenysVuika what do u guys think?

Reply:
what about centralizing around the "ws-notifications" as a top feature with some params, like:

```
{
  "notifications": {
    "task-filters": true,
    "process-filters": true
}
```

it's more like iOS/Android notification center settings as for me.

Also, we should not embed the technology stack into names, ws (web sockets) should not be in the name, what if we use different stack in the future... should be just "notifications" as for me

Reply:
I like that @DenysVuika, It'll allow for future expandability

Reply:
I agree to have a general notification property but I don't see to point of having the ability to turn one on and another off. We should probably call the property _**web-socket-notification**_